src/plugins/profiling/server/routes/flamechart.ts

```diff
@@ -200,206 +137,22 @@ async function queryFlameGraph(
   }
 
   // profiling-stacktraces is configured with 16 shards
-  const nQueries = 1;
-  const stackTraces = new Map<StackTraceID, StackTrace>();
-  const stackFrameDocIDs = new Set<string>(); // Set of unique FrameIDs
-  const executableDocIDs = new Set<string>(); // Set of unique executable FileIDs.
-  const stackTraceIDs = [...stackTraceEvents.keys()];
-  const chunkSize = Math.floor(stackTraceEvents.size / nQueries);
-  let chunks = chunk(stackTraceIDs, chunkSize);
-
-  if (chunks.length !== nQueries) {
-    // The last array element contains the remainder, just drop it as irrelevant.
-    chunks = chunks.slice(0, nQueries);
-  }
-
-  const stackResponses = await logExecutionLatency(
-    logger,
-    (testing ? 'search' : 'mget') + ' query for ' + stackTraceEvents.size + ' stacktraces',
-    async () => {
-      return await Promise.all(
-        chunks.map((ids) => {
-          if (testing) {
```

Comment:
Looks like you removed the code in the `testing` path. Can we keep it for a while ? The discussions haven't been settled yet.

Reply:
Oh, I moved the stacktrace query into `stacktrace.ts` so that it could be used by `flamechart.ts` and `topn.ts`. I split it into `searchStackTraces` and `mgetStackTraces` since the `testing` flag isn't used by the TopN response at the moment.

You can see the `flamechart.ts` calls here:
https://github.com/optimyze/kibana/pull/64/files#diff-ff74f07fae4cb5993bba147654de492b1e7fad820db1447c3b41dc138456b77aR140-R142

And the `topn.ts` call is here:
https://github.com/optimyze/kibana/pull/64/files#diff-b391fde7a8fee34bec4492c20662ec0d12bde2d3383b3c534a183885e9a8cec5R96-R101